docs/release-notes/8.4.asciidoc

```diff
@@ -9,6 +9,41 @@
 [[known-issue-8.4.2]]
 ==== Known issues
 * A new Lucene 9 validation change may cause event correlation rule (EQL) errors whenever rule queries contain regular expressions using wildcard fields and predefined character classes (for example, `\w`, `\s`, `\d`).
+* In some situations, {elastic-endpoint} might change to a non-running state on Windows endpoints and cannot be restarted. This failure might be also missed by {agent}, which would automatically re-install {elastic-endpoint} (https://github.com/elastic/endpoint/issues/29[#29]).
++
+To determine whether {elastic-endpoint} has stopped running, run the following PowerShell command as an administrator:
+
++
+[source,console]
+--------------------------------------------------
+PS C:\Users\user> Get-WinEvent Microsoft-Windows-CodeIntegrity/Operational | where Id -eq 3004 | where Message -match "elastic-endpoint.exe"
+
+
+   ProviderName: Microsoft-Windows-CodeIntegrity
+
+TimeCreated                      Id LevelDisplayName Message
+-----------                      -- ---------------- -------
+9/22/2022 10:47:35 AM          3004 Error            Windows is unable to verify the image integrity of the file \Device\HarddiskVolume3\Program Files\Elastic\Endpoint\elastic-endpo...
+9/19/2022 2:10:14 PM           3004 Error            Windows is unable to verify the image integrity of the file \Device\HarddiskVolume3\Program Files\Elastic\Endpoint\elastic-endpo...
+--------------------------------------------------
+
++
+
+If {elastic-endpoint} is not running, there are several workarounds you can take:
+
+** **Manually uninstall {elastic-endpoint} on affected hosts**: Remove an invalid {elastic-endpoint} install by manually invoking the {elastic-endpoint} <<uninstall-endpoint,uninstall command>> on affected hosts. Once the uninstallation process has finished, use the following command to restart {agent} and automatically re-install {elastic-endpoint}:
```

Comment:
```suggestion
** **Manually uninstall {elastic-endpoint} on affected hosts**: Remove an invalid {elastic-endpoint} install by manually invoking the {elastic-endpoint} <<uninstall-endpoint,uninstall command>> on affected hosts. Once the uninstallation process has finished, use the following command to restart {agent} which will automatically re-install {elastic-endpoint}:
```

Reply:
@ferullo would it be accurate to say that restarting the agent _also_ auto installs Endpoint?

```suggestion
** **Manually uninstall {elastic-endpoint} on affected hosts**: Remove an invalid {elastic-endpoint} install by manually invoking the {elastic-endpoint} <<uninstall-endpoint,uninstall command>> on affected hosts. Once the uninstallation process has finished, use the following command to restart {agent} and automatically re-install {elastic-endpoint}:
```

Reply:
yeah your edit looks good